### DISCUSSION TITLE
Dependabot PR commit messages include HTML version of PR description

### DISCUSSION THREAD

**Discussion by mhowardISP** [dependabot/dependabot-core#10229] [Q&A]
In one of our repositories the default commit message for PRs is to use `Pull request title and description`. For the Dependabot PRs the commit message has the HTML version of the description. Is it possible to modify what the Dependabot commit messages look like? If this is a bug I can open a bug instead.

**Comment by mofojed** [comment]
We're looking for this as well, looks like there is a feature request open to provide options for formatting: https://github.com/dependabot/dependabot-core/issues/1986
